app/build.gradle

```diff
@@ -47,10 +43,15 @@ license {
     strictCheck true
 }
 
+ext {
+    android_support_library = "26.1.0"
+}
+
+
 dependencies {
-    compile 'com.android.support:appcompat-v7:25.3.1'
-    compile 'com.android.support:recyclerview-v7:25.3.1'
-    compile 'com.android.support:design:25.3.1'
-    compile 'com.feedhenry:fh-android-sdk:3.3.3'
-    compile 'com.afollestad.material-dialogs:core:0.9.0.1'
+    implementation "com.android.support:appcompat-v7:$android_support_library"
```

Comment:
in principle I like the `$android_support_library` approach but it makes outdated versions less easy to spot in Android Studio

Reply:
For Android Studio it's should be the same, it is (or should be) smart enough to detect that. If not you can rebuild/update project and it will be detected.